library/alloc/src/collections/btree/node.rs

```diff
@@ -1737,6 +1722,28 @@ unsafe fn slice_remove<T>(slice: &mut [MaybeUninit<T>], idx: usize) -> T {
     }
 }
 
+/// Shifts the elements in a slice `distance` positions to the left.
+///
+/// # Safety
+/// The slice has at least `distance` elements.
+unsafe fn slice_shl<T>(slice: &mut [MaybeUninit<T>], distance: usize) {
+    unsafe {
```

Comment:
`warning: unnecessary 'unsafe' block` (same below)

Reply:
There's a `#![deny(unsafe_op_in_unsafe_fn)]` in library/alloc/src/lib.rs that would make that an error, but there is no such issue because `ptr::copy` is unsafe. How are you building this?

Reply:
Hmm, I think it must've been some incremental compilation glitch, because when I re-ran the compilation clean, everything went through; sorry to bother!